spec/spec_helpers/user_steps.rb

```diff
@@ -1,4 +1,4 @@
-def create_normal_user(id=random_id_generator)
+def create_normal_user(id = random_id_generator)
   @normal_user = User.new
   @normal_user.id = id
```

Comment:
Don't self assign IDs. The database will do that for you.

Also, please use factories instead of this rather manual setup.

Reply:
Okey dokey. Done.

One question: I've been having the problem often that when I run all my specs, my Factories cause validation errors - especially when making use of `traits`. For example, with `tags.` I need to use my `tag` factory for my `exercise`, `quotation` and `doctrine` specs. When I create a tag `trait` in more than one of these factories, it causes a validation error because the tag attribute `name` is already taken. 

How do I properly make re-use of the `tag` `trait` across multiple factories?